cnf-certification-test/suite_test.go

```diff
@@ -131,7 +117,30 @@ func TestTest(t *testing.T) {
 
 	// Set up logging params for logrus
 	loghelper.SetLogFormat()
-	setLogLevel()
+
+	// Delete execution log (if any) prior to run
+	logFile := "tnf-execution.log"
```

Comment:
As Ramon said, this file won't exist outside the container, unless you create it in the same folder as the claim file (see var `claimPath ` in the `init()` function of this file), as that folder is mounted into a local folder in the host using the "-v param of docker/podman. I suggest (maybe in a further PR) changing/unifying those flags to a single "--output" or "--output-folder" and use it for all the output files: junit, claimfile and this new execution log file.

Reply:
Yup I think I've almost got it fixed.

Reply:
@ramperher Can you re-run it with my latest changes?

Reply:
now the file is captured :) you can see it here: https://www.distributed-ci.io/jobs/e51cb6a8-2465-45ef-bb14-a8917107d65a/files